sql/synth_recipes.sql

```diff
@@ -3581,21 +3581,21 @@ INSERT INTO `synth_recipes` VALUES (55524,0,0,0,0,0,0,0,117,0,0,4099,4241,9003,9
 -- INSERT INTO `synth_recipes` VALUES (59918,1,0,0,0,0,0,0,255,0,0,4100,4242,16420,0,0,0,0,0,0,0,881,893,932,932,1,1,1,1,'Bone Patas (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59919,1,0,0,0,0,0,0,255,0,0,4100,4242,13199,0,0,0,0,0,0,0,817,924,893,893,1,1,1,1,'Blood Stone (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59920,1,0,0,0,0,0,0,255,0,0,4100,4242,17344,0,0,0,0,0,0,0,650,880,880,880,1,1,1,1,'Cornette (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59921,1,0,0,0,0,0,0,255,0,0,4100,4242,15159,0,0,0,0,0,0,0,506,1163,887,887,1,1,2,2,'Igqira Tiara (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59921,1,0,0,0,0,0,0,91,0,0,4100,4242,15159,0,0,0,0,0,0,0,506,1163,887,887,1,1,2,2,'Igqira Tiara (desynth)'); --yield mismatch
 -- INSERT INTO `synth_recipes` VALUES (59922,1,0,0,0,0,0,0,255,0,0,4100,4242,13110,0,0,0,0,0,0,0,818,893,893,893,1,1,1,1,'Beast Whistle (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59923,1,0,0,0,0,0,0,255,0,0,4100,4242,12451,0,0,0,0,0,0,0,896,851,851,851,1,1,1,1,'Scorpion Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59924,1,0,0,0,0,0,0,255,0,0,4100,4242,13312,0,0,0,0,0,0,0,744,887,887,887,1,1,1,1,'Coral Earring (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59924,1,0,0,0,0,0,0,83,0,0,4100,4242,13312,0,0,0,0,0,0,0,744,887,887,887,1,1,1,1,'Coral Earring (desynth)'); --yield mismatch
 -- INSERT INTO `synth_recipes` VALUES (59925,1,0,0,0,0,0,0,255,0,0,4100,4242,12717,0,0,0,0,0,0,0,816,850,1473,1473,3,2,1,1,'Scp. Gnt. +1 (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59926,1,0,0,0,0,0,0,255,0,0,4100,4242,13711,0,0,0,0,0,0,0,881,848,848,848,1,1,1,1,'Carapace Mask (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59926,1,0,0,0,0,0,0,45,0,0,4100,4242,13711,0,0,0,0,0,0,0,881,848,848,848,1,1,1,1,'Carapace Mask (desynth)'); --yield mismatch
 -- INSERT INTO `synth_recipes` VALUES (59927,1,0,0,0,0,0,0,255,0,0,4100,4242,13922,0,0,0,0,0,0,0,850,886,902,902,1,1,1,1,'Demon Helm (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,255,0,0,4100,4242,13442,0,0,0,0,0,0,0,864,888,888,888,1,1,1,1,'Shell Ring (desynth)');
+INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,7,0,0,4100,4242,13442,0,0,0,0,0,0,0,864,888,888,888,1,1,1,1,'Shell Ring (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59929,1,0,0,0,0,0,0,255,0,0,4100,4242,13461,0,0,0,0,0,0,0,881,1226,653,653,1,4,1,1,'Carapace Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59930,1,0,0,0,0,0,0,255,0,0,4100,4242,12455,0,0,0,0,0,0,0,852,852,852,852,1,1,1,1,'Beetle Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59931,1,0,0,0,0,0,0,255,0,0,4100,4242,14305,0,0,0,0,0,0,0,1629,1629,1629,1629,1,1,1,1,'Dragon Subligar (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59930,1,0,0,0,0,0,0,28,0,0,4100,4242,12455,0,0,0,0,0,0,0,852,852,852,852,1,1,1,1,'Beetle Mask (desynth)'); --yield mismatch
```

Comment:
Props for _not_ falling for the "Bonecraft Kit 30" trap that some BGWiki editor did for the synth. 😉